ui/apps/platform/cypress/helpers/wizard.ts

```diff
@@ -0,0 +1,15 @@
+export function getWizardNavStep(step: number | string) {
```

Comment:
Observation not change request, for generic helpers.
* Potential need to support exact (versus partial method) in the future is positive reason for `contains` method.
* IF a test needed to assert absence of an element, which would have been a potential improvement for conditional rendering in form tests, return of a selector instead of an element is more flexible. That is, cy.get` with chained methods might throw an error before caller has opportunity to call `should('not.exist')` method.
    Disclaimer: form helper methods have imperative `invoke` and `then` methods.
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/helpers/formHelpers.js#L1-L8